pycon-kr-2015/videos/pycon-kr-2015-lighting-talk-1st-day-raiteuning-tokeu-ceosjjaenal-20150830.json

```diff
@@ -7,7 +7,7 @@
     "Various speakers"
   ],
   "thumbnail_url": "https://i.ytimg.com/vi/p0ClB3w6w-o/hqdefault.jpg",
-  "title": "Lightning Talk",
+  "title": "Lightning Talk 1st day",
```

Comment:
'Lightning Talk 1st day' is unique for now. You can add the day or the conference to make it unique.